src/vs/monaco.d.ts

```diff
@@ -2910,6 +2910,10 @@ declare namespace monaco.editor {
 		 * The length of the range that got replaced.
 		 */
 		readonly rangeLength: number;
+		/**
+		 * The new end position of the range that got replaced.
+		 */
+		readonly rangeEndPosition: Position;
```

Comment:
Needed here, as the at the time that I was getting the position from the offset, the model could have changed and the position could be different.